segments/validators.go

```diff
@@ -52,29 +51,16 @@ func (v *validator) isPhoneNumber(number int64) error {
 	return nil
 }
 
-func (v *validator) filledString(s string) (string, error) {
-	if strings.Count(s, zeroString) == len(s) {
-		return zeroString, nil
-	}
-	if strings.Count(s, nineString) == len(s) {
-		return nineString, nil
-	}
-	if strings.Count(s, blankString) == len(s) {
-		return blankString, nil
-	}
-	return "", errors.New("not filled")
-}
-
 func (v *validator) isValidType(elm field, data string) error {
 	if elm.Required == required {
 		if elm.Type&numeric > 0 {
 			val, _ := strconv.Atoi(data)
 			if val == 0 {
-				return ErrRequired
+				return ach.ErrFieldRequired
 			}
-		} else if elm.Type&alphanumeric > 0 || elm.Type&alpha > 0 || elm.Type&binaryDescriptor > 0 {
+		} else if elm.Type&alphanumeric > 0 || elm.Type&alpha > 0 || elm.Type&descriptor > 0 {
 			if len(data) == 0 {
-				return ErrRequired
+				return ach.ErrFieldRequired
```

Comment:
Remove the dependency on the ACH project and copy over the code to Metro2

Reply:
fixed